src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -317,8 +317,8 @@ public async Task<IActionResult> GetScaleStatus([FromBody] ScaleStatusContext co
             }
 
             // TEMP: Once https://github.com/Azure/azure-functions-host/issues/5161 is fixed, we should take
-            // FunctionsScaleManager as a parameter.
-            if (Utility.TryGetHostService(scriptHostManager, out FunctionsScaleManager scaleManager))
+            // ScaleManager as a parameter.
+            if (Utility.TryGetHostService(scriptHostManager, out IScaleStatusProvider scaleManager))
```

Comment:
naming/comments no longer match type

Reply:
nit: one more in Line 330: FunctionsScaleManager does not exist anymore

Reply:
Change the name of the parameter to match as well - "scaleStatusProvider"